### ISSUE TITLE
Gray platelet

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4825]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
see https://www.nejm.org/doi/10.1056/NEJMoa1308130?url_ver=Z39.88-2003&rfr_id=ori%3Arid%3Acrossref.org&rfr_dat=cr_pub%3Dwww.ncbi.nlm.nih.gov

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
This is a synonym of https://hpo.jax.org/app/browse/term/HP:0012526, no need to change anything.
